### PR TITLE
Proofread docs

### PR DESCRIPTION
**Pull request by michaelclarkcuadrado**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -327,9 +508,9 @@ Filter | Description | Type | Default |
 `countries` | A list of 2 letter [ISO-2 country code](https://www.iso.org/iso-3166-country-codes.html) to limit matches. Example: `["US","BR","BE","AU"]` | array | null
 `indicators` | A string array of indicators to filter by. See [Available Sigma Indicators](#indicator-categories) to retrieve the values via API | string | null
```
Comment (olavoholanda) [inline source/index.html.md:509]:
```suggestion
`indicators` | A string array of indicators to filter by. See [Available Sigma Indicators](#indicator-categories) to retrieve the values via API | array | null
```

File: source/index.html.md [hunk 2]
```diff
@@ -327,9 +508,9 @@ Filter | Description | Type | Default |
...
 `dob` | A string representation of a date of birth. Two date formats are accepted: 1) `YYYY-MM-DD` or 2) `YYYY`<br/> | string | null
-`preferred_publishers` | A boolean value to restrict the news only for preferred publishers on the search | boolean | false
-`languages` | A comma separated list of languages to filter by. See [Available Languages](#languages) | string | null
-`events` | A comma separated list of events to filter by. See [Available Events](#events) | string | null
+`preferred_publishers` | A boolean value that restricts news media articles to well-vetted preferred publishers. See [Preferred Publishers](#preferred-publishers) for a full list. | boolean | false
+`languages` | An array of of languages to restrict news media articles to. See [Available Languages](#languages) | array | null
+`events` | A comma separated list of events to restrict news media articles to. See [Available Events](#events) | string | null
```
Comment (olavoholanda) [inline source/index.html.md:513]: this should be an array too, right?

File: source/index.html.md [hunk 3]
```diff
@@ -738,9 +860,9 @@ Filter | Description | Type | Default |
 `countries` | A list of 2 letter [ISO-2 country code](https://www.iso.org/iso-3166-country-codes.html) to limit matches. Example: `["US","BR","BE","AU"]` | array | null
 `indicators` | A string array of indicators to filter by. See [Available Sigma Indicators](#indicator-categories) to retrieve the values via API | string | null
```
Comment (olavoholanda) [inline source/index.html.md:861]: array

File: source/index.html.md [hunk 4]
```diff
@@ -738,9 +860,9 @@ Filter | Description | Type | Default |
...
 `dob` | A string representation of a date of birth. Two date formats are accepted: 1) `YYYY-MM-DD` or 2) `YYYY` | string | null
-`preferred_publishers` | A boolean value to restrict the news only for preferred publishers on the search | boolean | false
-`languages` | A comma separated list of languages to filter by. See [Available Languages](#languages) | string | null
-`events` | A comma separated list of events to filter by. See [Available Events](#events) | string | null
+`preferred_publishers` | A boolean value that restricts news media articles to well-vetted preferred publishers | boolean | false
+`languages` | A comma separated list of languages to restrict news media articles to. See [Available Languages](#languages) | string | null
+`events` | A comma separated list of events to restrict news media articles to. See [Available Events](#events) | string | null
```
Comment (olavoholanda) [inline source/index.html.md:865]: arrays

File: source/index.html.md [hunk 5]
```diff
@@ -1063,9 +1126,9 @@ Parameter | Description | Type | Default
 `mode` | *DEPRECATED* Sigma integrations filter enables configuration of which integrations are used and how the data is returned | string | basic
 `indicators` | A comma separated list of indicators to filter by. See [Available Sigma Indicators](#indicator-categories) to retrieve the values via API | string | empty
```
Comment (olavoholanda) [inline source/index.html.md:1127]: array

File: source/index.html.md (hunk 4 above)
Comment (michaelclarkcuadrado) [reply source/index.html.md:865]: Fixed

File: source/index.html.md (hunk 2 above)
Comment (michaelclarkcuadrado) [reply source/index.html.md:513]: Fixed

**Pull request merged**